test/test_regression_cli.py

```diff
@@ -1,34 +1,54 @@
-from __future__ import print_function
```

Comment:
@goelakash, yes I have seen that even in json some imports are missing, I do have them on my system, I have no idea how they have been removed. when I committed. Lately my python environment had been a stranger to me :)